### DISCUSSION TITLE
tutorial in Gromacs, example 4

### DISCUSSION THREAD

**Discussion by xih72** [SIRAHFF/documentation#75] [Q&A] [answered]
https://sirahff.github.io/documentation/Tutorials%20gromacs.html#closed-circular-dna-in-explicit-solvent
4.2. PDB to GROMACS format
gmx pdb2gmx -f ccdna_cg.pdb -o ccdna_cg.gro
-------------------------------------------------------
Program:     gmx pdb2gmx, version 2018
Source file: src/gromacs/gmxpreprocess/pdb2gmx.cpp (line 753)

Fatal error:
Atom PX in residue DCX 1 was not found in rtp entry CX5 with 5 atoms
while sorting atoms.
.

For more information and tips for troubleshooting, please check the GROMACS
website at http://www.gromacs.org/Documentation/Errors
-------------------------------------------------------

**Comment by SIRAHFF** [comment, marked as answer]
You got this error because of a problem with the spaces in the previous command. 

We changed it in the documentation. It should work now.

**Comment by xih72** [comment]
It works now. You can close this ticket.
